### PR TITLE
Add tests for issues

### PR DESCRIPTION
**Pull request by birdstorm**
add tests for #160

### CODE REVIEW

File: integtest/src/main/scala/com/pingcap/spark/IssueTestCase.scala [hunk 1]
```diff
@@ -39,17 +39,64 @@ class IssueTestCase(prop: Properties) extends TestCase(prop) {
...
+    result |= execAllAndJudge("select * from t order by c1, c2")
+    result |= execAllAndJudge("select * from t order by c1 asc, c2 asc")
+    result |= execAllAndJudge("select * from t order by c1 asc, c2 desc")
+    result |= execAllAndJudge("select * from t order by c1 desc, c2 asc")
+    result |= execAllAndJudge("select * from t order by c1 desc, c2 desc")
+    result |= execAllAndJudge("select * from t order by (c1, c2)")
+    result |= execAllAndJudge("select * from t order by (c1, c2) asc")
+    result |= execAllAndJudge("select * from t order by (c1, c2) desc")
+    result = !result
+    logger.warn(s"\n*************** Issue Spark#0162 Tests result: $result\n\n\n")
```
Comment (Novemser) [inline integtest/src/main/scala/com/pingcap/spark/IssueTestCase.scala]: 162->160

**Comment by Novemser** [comment]
Issue stated in #160 has not been fixed.

File: integtest/src/main/scala/com/pingcap/spark/IssueTestCase.scala (hunk 1 above)
Comment (birdstorm) [reply integtest/src/main/scala/com/pingcap/spark/IssueTestCase.scala]: thx for pointing oit

Comment (Novemser) [reply integtest/src/main/scala/com/pingcap/spark/IssueTestCase.scala]: BTW, coding on the subway might not be a good habit. LOL :laughing:

**Comment by birdstorm** [comment]
@Novemser ok I will delete it, but we must enclose this test in the corresponding pull request then.

**Comment by birdstorm** [comment]
@Novemser PTAL.

**Comment by Novemser** [comment]
LGTM

**Pull request merged**